docs/Development/ReleaseNotes.md

```diff
@@ -10,13 +10,13 @@ For a list of known issues and their fixes (in bug-fix updates), visit the VIC G
 
 To check which release of VIC you are running:
 
-Type `vicNl -v` or for VIC 5 and later, `vic_{driver}.exe -v`.
+Type `vicNl -v` or for VIC 5 and later, `vic_{classic,image}.exe -v`.
```

Comment:
Because of comma placement, order, and expectation related to which version of VIC someone reading this would have, on a fast read/skim, I think that `vicNl -v` is for VIC 5 and later. Maybe write:
For VIC 4, type `vicNl -v`.
For VIC 5 and later, type `vic _{classic,image}.exe -v`.